src/python/pants/backend/visibility/glob.py

```diff
@@ -28,17 +28,21 @@ class PathGlob:
     raw: str
     anchor_mode: PathGlobAnchorMode = field(compare=False)
     glob: Pattern = field(compare=False)
+    uplvl: int
```

Comment:
this is to support rules like:
```python
("*", "../../some/other/sibling/tree/**", ...)
```

Reply:
currently only for "invoked" anchored paths, i.e. those that start with a `.`.. should perhaps warn in case it is used in other modes, like for `/../../some/path/*`